### PR TITLE
Update extract-sv-reads to v1.2.1

### PR DESCRIPTION
**Pull request by ernfrid**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This updates extract-sv-reads to v1.2.1. See the release notes for details.

### CODE REVIEW

File: recipes/extract-sv-reads/meta.yaml [hunk 1]
```diff
@@ -4,22 +4,26 @@ build:
...
-    - zlib
+    - curl
+    - bzip2 {{ CONDA_BZIP2 }}*
+    - xz
   run:
     - libgcc
-    - zlib
+    - curl
+    - bzip2 {{ CONDA_BZIP2 }}*
+    - xz
```
Comment (bgruening) [inline recipes/extract-sv-reads/meta.yaml]: can we pin xz 5.2.* as well

Comment (ernfrid) [reply recipes/extract-sv-reads/meta.yaml]: Sure thing. What's required to do that? Add a line to env_matrix.yml like so?
```
CONDA_XZ: "5.2"
```

Comment (bgruening) [reply recipes/extract-sv-reads/meta.yaml]: Yes, this would be supercool!

**Comment by ernfrid** [comment]
Updated as requested. Let me know if this addresses your comments @bgruening

**Comment by bgruening** [comment]
Fantastic! :)

**Pull request merged**